lib/vagrant-service-manager/binary_handlers/binary_handler.rb

```diff
@@ -100,7 +100,7 @@ def validate_url
         req.use_ssl = true if url.scheme == 'https'
         res = req.request_head(url.path)
 
-        unless res.code == '200'
+        unless %w(200 302).include? res.code
```

Comment:
Again, that's why we need a verified end to end test. TBH, I am surprised. I thought I tested it, but maybe it was only 'docker' after all.